ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -695,6 +699,221 @@ literals:
+  - ACTIVE
+  - INACTIVE
+- low_compressor_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- compressor_current_alarm:
+  - ACTIVE
+  - INACTIVE
+- compressor_run_time_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_high_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_overflow_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_low_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- fan_speed_frequency_setpoint
+- compressor_speed_frequency_setpoint
+- start_mode:
+  - ON
+  - OFF
+- reset_alarm:
+  - ACTIVE
+  - INACTIVE
+- reset_command:
+  - ON
+  - OFF
+- chilled_water_isolation_valve_mode:
+  - AUTO
+  - MANUAL
+- pump_speed_frequency_setpoint
+- chilled_water_isolation_valve_percentage_setpoint
+- chilled_water_flowrate_alarm:
+  - ACTIVE
+  - INACTIVE
+- end_alarm:
+  - ACTIVE
+  - INACTIVE
+- start_alarm:
+  - ACTIVE
+  - INACTIVE
+- simulation_command:
+  - ON
+  - OFF
+- condensing_supply_water_isolation_simulation_command:
+  - ON
+  - OFF
+- stop_command:
+  - ON
+  - OFF
+- closed_command:
+  - ON
+  - OFF
+- closed_alarm:
+  - ACTIVE
+  - INACTIVE
+- open_alarm:
+  - ACTIVE
+  - INACTIVE
+- simulation_status:
+  - ON
+  - OFF
+- local_status:
+    - REMOTE
+    - LOCAL
+- compressor_open_percentage_command
+- high_pressure_sensor
+- low_pressure_sensor
+- high_temperature_sensor
+- low_temperature_sensor
+- compressor_current_sensor
+- high_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- high_temperature_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_temperature_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_water_differential_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- chilled_supply_water_low_temperature_alarm:
+  - ACTIVE
+  - INACTIVE
+- chilled_water_differential_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_water_flowrate_alarm:
+  - ACTIVE
+  - INACTIVE
+- current_alarm:
+  - ACTIVE
+  - INACTIVE
+
+- motor_energy_sensor
+- motor_frequency_sensor
+- heat_percentage_sensor
+- fan_voltage_sensor
+- break_time_command
+- fin_temperature_sensor
+- heat_inverter_percentage_sensor
+- electric_inverter_percentage_sensor
+- max_inverter_current_sensor
+- condensing_return_water_isolation_control_status:
+  - LOCAL
+  - REMOTE
+- condensing_return_water_isolation_closed_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_return_water_isolation_open_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_return_water_isolation_reset_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_return_water_isolation_simulation_command:
+  - ON
+  - OFF
+- condensing_return_water_isolation_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_supply_water_isolation_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_supply_water_isolation_closed_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_supply_water_isolation_open_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_supply_water_isolation_reset_alarm:
+  - ACTIVE
+  - INACTIVE
+- open_command:
+  - ON
+  - OFF
+- condensing_supply_water_isolation_valve_run_command:
+  - ON
+  - OFF
```

Comment:
i am not convinced any of these are needed; some are malformed; some seem unnecessary outside the BMS. what is the use case for them?

Reply:
- condensing_return_water_isolation_closed_alarm
- condensing_return_water_isolation_open_alarm
- condensing_return_water_isolation_reset_alarm
- condensing_return_water_isolation_simulation_command
- condensing_supply_water_isolation_closed_alarm
- condensing_supply_water_isolation_open_alarm
- condensing_supply_water_isolation_reset_alarm

The points stated above are not necessary so they are removed. The rest of the points should stay as they are isolation value points that can be used verify why chiller cannot run (if chiller is turned on by mistake)

Reply:
There is also no such point name called "condensing_supply_water_isolation_valve_run_command", so it is removed as well